index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="zxx">
+   <head>
+      <meta charset="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>ADOPTADOG</title>
+      <link rel="stylesheet" type="text/css" href="styles/stylesheet.css" />
+      <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+      <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Roboto'>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+      <link href='https://fonts.googleapis.com/css?family=Arvo' rel='stylesheet'>
+   </head>
+   <body>
+      <header>
+         <div class="header">
+            <div class="header-logo">
+               <img src="images/adoptadog-logo.svg" alt="Image of the Company Logo" />
+               <h2 class="slogan">Seattle's premiere dog adoption agency</h2>
+            </div>
+            <div class="shoppingcart">
+               <img src="images/cart-icon.svg" alt="Image of the Cart" />
+               <p>$1234.56</p>
+               <button class="checkout-button" onclick="document.location='checkout.html'">Checkout</button>
+            </div>
+         </div>
+         <nav>
+            <ul>
+               <li><a href="index.html">Home </a> </li>
+               <li><a href="dogs.html"> Dogs </a> </li>
+               <li><a href="blog.html"> Blog </a> </li>
+            </ul>
+         </nav>
+      </header>
+      <main>
+         <div class="flex-container">
+            <div class="box">
+               <div class="container-about">
+                  <img src="images/home-intro.jpg" alt="Image of the home intro " />
+                  <h3 >Adopt A Dog</h3>
+                  <p>
+                     Iduciendisite quo magnatem iuntum quid quaest ea am, <br>tenderumet adis dolenem quidustrum fuga. Faceaquae estioria <br> derum recuptatur, cum volore, undipsa doloreium hillupta aut <br>es ut alitatuscit ommossum
+                     haritatur arum qui officae videbiti <br>corporeium faccull oribus es quidignis ipietus explam sus <br> am aut amet ant fugiatum, utem non reptat ad magnienecto <br> volorent fugia nonsedi gendand itatquos as magnimpor aut <br>
+                     molor autatia ad est officimus accae et omnia cus, od qui ate
+                  </p>
+               </div>
+            </div>
+         </div>
+         <div class="grid-container">
```

Comment:
Your grid should have the same left-margin as the image above it. You want the golden retriever photo to be aligned with your grid.